5-VM-Ubuntu-1-NodeTypes-Secure/azuredeploy.json

```diff
@@ -1,39 +1,40 @@
 {
-  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
   "contentVersion": "1.0.0.0",
   "parameters": {
-    "location": {
+    "clusterLocation": {
       "type": "string",
-      "defaultValue": "[resourceGroup().location]",
+      "defaultValue": "westus",
```

Comment:
leave this set to [resourceGroup().location] otherwise defaults tend to overload a datacenter

Reply:
Done. Being a template I don't understand how it would overload a datacenter? Only presumption is customers may commonly use template values if their use case doesn't require them to change it, and as such a specific location value may result in customer's deciding to overload that specific location by accepting the template value?

Reply:
exactly